docs/en/faq/why-kotlin-jdsl-returns-nullable-list-in-the-spring-support-module.md

```diff
@@ -0,0 +1,39 @@
+# Why does Kotlin JDSL allow nullable return types?
+
+Kotlin JDSL helps developers develop more efficiently by allowing them to write queries without creating a separate metadata model or making string syntax errors.
+However, Kotlin JDSL users may sometimes have questions about nullable return types in the form of Slice<T?> or Page<T?> returned when calling APIs like findSlice or findPage methods.
+
+## Why do we need nullable return types?
+
+The reason why Kotlin JDSL supports nullable return types is that depending on how the JPQL query is written, some of the returned list items may contain null values.
+The simplest example is when you look up a column or entity without using DTO Projection.
+
+The table below shows some of the cases where nullable return types can occur when using the Kotlin JDSL.
+
+| Item           | Nullable or not | Reason                                                                                               |
+|----------------|-----------------|------------------------------------------------------------------------------------------------------|
+| DTO Projection | X               | Calling the constructor for all ROWs results in the object being created, which does not allow nulls |
+| Column         | O               | Exists when looking up a field that is null                                                          |
+| Entity         | O               | Exists if the Entity being joined on Left Join is null.                                              |
+
+As another example, the code below shows a situation where the Author entity exists, but the BookAuthor entity, the target of the left join, may be null.
+
+```kotlin
+val query = jpql {
+    select(
+        path(BookAuthor),
```

Comment:
```suggestion
        path(Author::BookAuthor),
```
I guess this code is probably correct.

Reply:
Thanks for letting us know, I've fixed it!